kernel/src/main/java/org/kframework/EvaluationException.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2018 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+package org.kframework;
+
+import java.util.ArrayList;
+import org.kframework.kore.K;
+
+
+public class EvaluationException extends RuntimeException {
+    private int steps_;
```

Comment:
also, you don't need the underscore. You can just say this.steps = steps; in the constructor.